HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/MainActivity.kt

```diff
@@ -0,0 +1,239 @@
+                    } else {
+                        Handler(Looper.getMainLooper()).post { binding.btnShow.isEnabled = true }
+                        addToLogView("$placementName(HeliumInterstitialAd) didCache")
+                    }
+                }
+
+                override fun didShow(placementName: String, error: HeliumAdError?) {
+                    if (error != null) {
+                        addToLogView(placementName + "(HeliumInterstitialAd) didShow failed with error: " + error.getMessage())
+                    } else {
+                        addToLogView("$placementName(HeliumInterstitialAd) didShow")
+                    }
+                }
+
+                override fun didClose(placementName: String, error: HeliumAdError?) {
+                    if (error != null) {
+                        addToLogView(placementName + "(HeliumInterstitialAd) didClose failed with error: " + error.getMessage())
+                    } else {
+                        addToLogView("$placementName(HeliumInterstitialAd) didClose")
+                    }
+                }
+            })
+        return interstitialAd
+    }
+
+    private fun createRewardedAd(): HeliumRewardedAd {
+        val rewardedPlacement =
+            binding.rewardedPlacementName.text?.toString() ?: getString(R.string.heliumRewarded)
+        val rewardedAd = HeliumRewardedAd(rewardedPlacement, object : HeliumRewardedAdListener {
+            override fun didReceiveWinningBid(
+                placementName: String,
+                bidInfo: HashMap<String, String>
+            ) {
+                addToLogView("$placementName(HeliumRewardedAd) didReceiveWinningBid")
+                addToLogView(bidInfo.toString())
+            }
+
+            override fun didCache(placementName: String, error: HeliumAdError?) {
+                if (error != null) {
+                    addToLogView(placementName + "(HeliumRewardedAd) didCache failed with error: " + error.getMessage())
+                } else {
+                    Handler(Looper.getMainLooper()).post {
+                        binding.btnShowRewarded.isEnabled = true
+                    }
+                    addToLogView("$placementName(HeliumRewardedAd) didCache")
+                }
+            }
+
+            override fun didShow(placementName: String, error: HeliumAdError?) {
+                if (error != null) {
+                    addToLogView(placementName + "(HeliumRewardedAd) didShow failed with error: " + error.getMessage())
+                } else {
+                    addToLogView("$placementName(HeliumRewardedAd) didShow")
+                }
+            }
+
+            override fun didClose(placementName: String, error: HeliumAdError?) {
+                if (error != null) {
+                    addToLogView(placementName + "(HeliumRewardedAd) didClose failed with error: " + error.getMessage())
```

Comment:
Did you want to kotlinize these statements? 
```
error?.let {
    addToLogView("$placementName etc)
}?: addToLogView("other message")
```

Reply:
I've tested and with the indentation looks very ugly and difficult to read, I think is more clear the current way

Reply:
fair enough. what about the kotlin standard string builder using `$placementName`?

Reply:
for sure!